### DISCUSSION TITLE
Hey man, just noticed something that's kinda triggering me on your portfolio.

### DISCUSSION THREAD

**Discussion by cakePhone** [community/community#67861] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I've noticed your portfolio has unnecessary scrolls on some pages, mainly the home and portfolio pages with a little scroll despite there being space to display all the info. I was looking into your github to see if there was a public repo for that website so I could do those fixes for you and just do a pr, but there isn't one. So I came here to submit my little "complaint" about it. Thank you! Have a nice day! Happy coding.

**Comment by adamlui** [comment]
Hey nice tie
